database/gormdb_tasks.go

```diff
@@ -98,3 +99,45 @@ func (db *GormDB) SynchronizeAssignmentTasks(course *pb.Course, taskMap map[uint
 
 	return createdTasks, updatedTasks, err
 }
+
+// CreatePullRequest creates a pull request
+func (db *GormDB) CreatePullRequest(pullRequest *pb.PullRequest) error {
+	if !pullRequest.Validate() {
+		return errors.New("pull request is not valid for creation")
+	}
+	pullRequest.SetDraft()
```

Comment:
Perhaps we should document in the comment and elsewhere that initially it is set to draft mode.